trade-mapping-to-ipaffs/src/test/java/uk/gov/defra/stw/mapping/toipaffs/integration/ChedpNotificationMapperIntegrationTest.java

```diff
@@ -29,35 +26,27 @@ class ChedpNotificationMapperIntegrationTest {
 
   @Test
   void map_ReturnsChedpNotification_WhenCompleteEhcSpsCertificate() throws Exception {
-    ObjectMapper objectMapper = TestUtils.initObjectMapper();
+    SpsCertificate spsCertificate = JsonDeserializer.get("chedp/chedp_ehc_complete.json",
+        SpsCertificate.class);
 
-    SpsCertificate spsCertificate =
-        JsonDeserializer.get(
-            SpsCertificate.class, "chedp/chedp_ehc_complete.json", objectMapper);
 
-    String expectedNotification =
-        ResourceUtils.readFileToString("classpath:chedp/chedp_ipaffs_complete.json");
+    Notification actual = chedpNotificationMapper.map(spsCertificate);
+    overrideUniqueComplementIdToStaticValue(actual);
 
-    Notification notification = chedpNotificationMapper.map(spsCertificate);
-    overrideUniqueComplementIdToStaticValue(notification);
-
-    String actualNotification = objectMapper.writeValueAsString(notification);
-
-    assertThat(actualNotification).isEqualTo(expectedNotification);
+    Notification expected = JsonDeserializer.get("chedp/chedp_ipaffs_complete.json",
+        Notification.class);
+    assertThat(actual)
```

Comment:
Changed to use recursive comparison instead of string comparison so whitespace in the JSON file is not taken into account - makes the test less brittle